files.sh

```diff
@@ -2,7 +2,7 @@
 
 fn nash_complete_paths(parts, line, pos) {
 	partsz   <= len($parts)
-	last     <= -expr $partsz - 1
+	last     <= -echo $partsz - 1 | bc
```

Comment:
When result of expression is 0, **expr** exit status is 1 and nash don't set **last** var.